torch/_dynamo/variables/builtin.py

```diff
@@ -1471,8 +1477,16 @@ def _unimplemented():
         if isinstance(left, UserDefinedObjectVariable) and isinstance(
             right, UserDefinedObjectVariable
         ):
-            return ConstantVariable.create(op(left.value, right.value))
-
+            ret = op(left.value, right.value)
```

Comment:
Let's remove this. We can handle generically in another way. This PR is also blocked on vmap FakeTensor identity.